### ISSUE TITLE
Size of the NFHL table

### ISSUE THREAD

**Issue by tph018** [seabre/fema-nfhl-postgis-load#1]
Hi,

I was briefly reviewing your code and was wondering how big in terms of bytes the entire NFHL table (the s_fld_haz_ar table) based on all U.S. states within PostGIS? Is the hazard table within PostGIS on the order of 50GB or less?

I am attempting to do a similar conversion using ogr2ogr of a more recent file NFHL_08082016.gdb by converting from this ESRI GDB directly into a PostGIS DB and after a half-day's worth of conversion, the files are not fully converted yet and I am wondering how much longer it shall take to complete the conversion/upload into PostGIS.

Thanks!
-tph018

**Comment by seabre** [comment]
A blast from the past!

If I remember correctly, when finished, the full data in the table was indeed around 50GB or so. Took a long time. Maybe a day with CPU @ 2.90GHz and 8GB of RAM on Ubuntu 14.04. I didn't have an SSD at the time, either, so that probably would have helped a lot.

Yeah, the download list is pretty old. I haven't updated this in a long time.

**Comment by tph018** [comment]
Thanks for the color! It is appreciated...

I had converted a full 40GB or so and had to abort the existing conversion process due to the long conversion time since the hazard file is on my local machine and the postgis server is on a network machine. Now I at least know roughly how much longer I would have had to go to get to the end (which seems to be about another 10GB or so or 20% left). I guess I can try the whole conversion again tomorrow...

Thanks again,
-tph018

**Comment by seabre** [comment]
Yeah honestly, if possible, I would just take a day and let it sit. Check in every 2 or 3 hours so if it fails you can see what went wrong and restart.
